### PR TITLE
Security considerations amended

### PR DESCRIPTION
**Pull request by henkvancann**
I added a reason to the list (compatibility with did:web), but not sure if that's valid.

Furthermore, I've suggested a few why's and how's of the list that you should maybe want to take a close look at.

### CODE REVIEW

**Comment by henkvancann** [comment]
1. it safeguards privacy (this is implicitly offered by the https protocol)
2. discourages denial of service attacks (HOW?), 
3. accords with a defense-in-depth mindset (WHY?), 
4. aids regulatory compliance (because we'd comply with common technologic practise),
5. establishes backwards compatibility with `did-web` (for we would use exactly the same form),
6. and makes it easier to troubleshoot (HOW/WHY?).

**Review by 2byrds — CHANGES_REQUESTED** [review]
Privacy/https fix is the biggest change needed.

File: spec/security_considerations.md [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
-cybersecurity best practice. This is not strictly necessary to guarantee the
-authenticity of the data. However, it safeguards privacy, discourages denial of
-service, accords with a defense-in-depth mindset, aids regulatory compliance,
-and makes it easier to troubleshoot. How a URL is hosted should let clients
-fetch a DID document and a [[ref: KERI event stream]] with roughly the same confidence that's associated
-with properly implemented online banking.
+cybersecurity best practice. Mind you, this **is not strictly necessary to guarantee the
+authenticity of the data**. However, there are several other reasons to stick to best practises around HTTPS and URLs, also for `did:webs`.
+
+1. it safeguards privacy (this is implicitly offered by the https protocol)
```
[the remark this answers is not in the document]
Comment (henkvancann) [reply spec/security_considerations.md:10]: I just regrouped formatted it in a list, did not introduce the word 'privacy'
but I do agree we need to reconsider the wording
Maybe we discuss this briefly next meeting?

**Comment by henkvancann** [comment]
> Privacy/https fix is the biggest change needed.

done

**Pull request closed without merging**